### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by haitonno** [spbrogan/mopeka_pro_check#4] [Q&A]
Hi Sean,

I'm trying to get this work, but still no success. I'm using a standard Model of the Sensor (1001 356B) and try to read this via Raspi Zero W. 
First question: How did you discover the Mac Adress of the Sensor. As I unserstand, this is needed to get the Sensor connected.
I only have the last 6 digits of the Adress.

Regards Rainer

**Comment by spbrogan** [comment]
This is kind of old and not sure if you are still stuck.  
There is a service you can call.
https://github.com/spbrogan/sensor.mopeka_pro_check#setting-up-a-new-sensor
